pkg/routeagent_driver/handlers/ovn/submarinerNonGWRouteHandler.go

```diff
@@ -0,0 +1,172 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ovn
+
+import (
+	"context"
+	"sync"
+
+	"github.com/pkg/errors"
+	submarinerv1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	submarinerClientset "github.com/submariner-io/submariner/pkg/client/clientset/versioned"
+	"github.com/submariner-io/submariner/pkg/event"
+	nodeutil "github.com/submariner-io/submariner/pkg/node"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+const (
+	ovnTransitSwitchIPAnnotation = "k8s.ovn.org/node-transit-switch-port-ifaddr"
+)
+
+type NonGatewayRouteHandler struct {
+	event.HandlerBase
+	mutex           sync.Mutex
+	smClient        submarinerClientset.Interface
+	remoteEndpoints map[string]*submarinerv1.Endpoint
+	k8sClientSet    clientset.Interface
+	isGateway       bool
+	transitSwitchIP string
+}
+
+func NewNonGatewayRouteHandler(smClientSet submarinerClientset.Interface, k8sClientset *clientset.Clientset) *NonGatewayRouteHandler {
+	// We'll panic if env is nil, this is intentional
+	return &NonGatewayRouteHandler{
+		smClient:        smClientSet,
+		remoteEndpoints: map[string]*submarinerv1.Endpoint{},
+		k8sClientSet:    k8sClientset,
+	}
+}
+
+func (nonGatewayRouteHandler *NonGatewayRouteHandler) Init() error {
+	logger.Infof("Starting NonGatewayRouteHandler")
```

Comment:
We can use "logger.Info" and at other places too.

```suggestion
	logger.Info("Starting NonGatewayRouteHandler")
```

Reply:
Done.